provider/schema/dialect.go

```diff
@@ -197,6 +197,10 @@ func (d TSDBDialect) GetResourceValues(r *Resource) ([]interface{}, error) {
 	return doResourceValues(d, r)
 }
 
+func (d TSDBDialect) DefaultSchema() string {
+	return "history"
```

Comment:
^ more botherage

Reply:
why do you need to add the default schema here? when running the migrations should be in the correct search path and they should be schema agnostic. Or maybe I missed something.

Reply:
@roneli because on TSDB the actual constraint isn't in the search path (`public`) as we have views in there. So list of PKs return 0 rows when queried with `tavle_schema='public'`